bcda/middleware_test.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/go-chi/chi"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type MiddlewareTestSuite struct {
+	suite.Suite
+	server *httptest.Server
+}
+
+func (s *MiddlewareTestSuite) SetupTest() {
+	router := chi.NewRouter()
+	router.With(ValidateBulkRequestHeaders).Get("/", func(w http.ResponseWriter, r *http.Request) {
+		_, err := w.Write([]byte("Test router"))
+		if err != nil {
+			log.Fatal(err)
+		}
+	})
+
+	s.server = httptest.NewServer(router)
+}
+
+func (s *MiddlewareTestSuite) TestValidateBulkRequestHeaders() {
+	client := s.server.Client()
+
+	req, err := http.NewRequest("GET", s.server.URL, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	req.Header.Add("Accept", "application/fhir+json")
+	req.Header.Add("Prefer", "respond-async")
+
+	resp, err := client.Do(req)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	assert.Equal(s.T(), 200, resp.StatusCode)
+}
+
+func (s *MiddlewareTestSuite) TestValidateBulkRequestHeadersInvalidAccept() {
+	client := s.server.Client()
+
+	req, err := http.NewRequest("GET", s.server.URL, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	req.Header.Add("Accept", "")
+
```

Comment:
Accept is invalid here, but Prefer is also invalid.  
Can you set Prefer to be valid, so that we guarantee the test is accurately testing the Accept scenario?